packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -291,12 +293,20 @@ export class TokenRatesController extends StaticIntervalPollingControllerV1<
   }
 
   /**
-   * Start (or restart) polling.
+   * Start (or resume) polling.
    */
   async start() {
```

Comment:
it looks like you can also start the controller via `startPollingByNetworkClientId`.  May need to handle that case.

Reply:
I may move this feature to `StaticIntervalPollingController` so other controllers can use it